framework/encode/dx12_state_tracker.cpp

```diff
@@ -653,7 +668,10 @@ void Dx12StateTracker::TrackBuildRaytracingAccelerationStructure(
                 ID3D12Resource_Wrapper* src_resource_wrapper = nullptr;
                 {
                     std::unique_lock<std::mutex> lock(state_table_mutex_);
-                    src_resource_wrapper = GetResourceWrapperForGpuVa(*curr_entry_iter->desc_gpu_va);
+                    src_resource_wrapper =
+                        GetResourceWrapperForGpuVa(*curr_entry_iter->desc_gpu_va,
+                                                   *curr_entry_iter->desc_gpu_va + curr_entry_iter->size,
+                                                   IsResourceUsedForAccelerationStructure);
```

Comment:
This looking for GPUVA does not expected to return AS resource. no need IsResourceUsedForAccelerationStructure .